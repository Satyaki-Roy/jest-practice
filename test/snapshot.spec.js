import { test } from "@jest/globals";

const user = { name: "john", age: 22, class: "BE" };

test("snapshot test", () => {
  expect(user).toMatchSnapshot();
});
