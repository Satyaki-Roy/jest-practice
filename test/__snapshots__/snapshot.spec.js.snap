// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`snapshot test 1`] = `
Object {
  "age": 22,
  "class": "BE",
  "name": "john",
}
`;
